### DISCUSSION TITLE
Alerting template for telegram is broken after upgrade to 10.0

### DISCUSSION THREAD

**Discussion by makavity** [grafana/grafana#71924] [Alerting]
Hello!
I'm using template like this:
```go
{{ define "tgshortbody" }}{{ range . }} <b>{{ or .Annotations.message .Labels.alertname }}</b>
{{ range .Annotations.SortedPairs }}{{ if ne .Name "message" }}{{ .Name }}: {{ .Value }}
{{ end }}{{ end }}
{{ with .ValueString }}{{ reReplaceAll "[[][^]]*metric='{?([^}']*)}?'[^]]*value=([0-9]*([.][0-9]{,3})?)[^]]*](, )?" "$1: <b>$2</b>\n" . }}
{{ end }}
{{ end }}{{ end }}
{{ define "tgshort" }}{{ with .Alerts.Firing }}❌{{ template "tgshortbody" . }}{{ end }}{{ with .Alerts.Resolved }}✅{{ template "tgshortbody" . }}{{ end }}{{ end }}
```

Before update to 10.0 (v. 9.5) alerts with classic condition displayed like this:
<img width="336" alt="image" src="https://github.com/grafana/grafana/assets/5614215/348a2780-b8e9-444b-b7ce-84d4421622d8">

Now, labels is not working for "metric" field. It is always "value".
<img width="299" alt="image" src="https://github.com/grafana/grafana/assets/5614215/118d6b0c-d942-4275-88ee-a25ed0ff0fd8">

Is it bug, or should I change my template for alerting?
Thx.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82622. Feel free to continue the discussion around this there. Thank you!

**Comment by armandgrillet** [comment]
You should just use `$value` instead of `.ValueString`, this was deprecated in Grafana 10.
